lib/erb_lint/cli.rb

```diff
@@ -113,6 +115,7 @@ def run_with_corrections(filename)
         end
 
         file_content = corrector.corrected_content
+        break if previous_offenses.eql?(runner.offenses)
```

Comment:
Won't the instance of this array always be equal to itself?

Reply:
You're right, I change the strategy to instead clear the ivars at the end of the loop

Reply:
🆒 since the list is append-only you could also look at the array size